Automattic/wp-calypso
# Color Schemes: Prep to launch Nightfall, Powder Snow, and Sakura color schemes.

#### Changes proposed in this Pull Request

* Let the color schemes fly! No more hiding them behind feature flags.
* Also removes the now unneeded config values for these color schemes.

#### Testing instructions

* Switch to this PR and navigate to `/me/account`. You should see Nightfall, Powder Snow, and Sakura in the Color Schemes section.
